Python/Product/PythonTools/PythonTools/Repl/LoadReplCommand.cs

```diff
@@ -39,10 +39,14 @@ class LoadReplCommand : IInteractiveWindowCommand {
         public Task<ExecutionResult> Execute(IInteractiveWindow window, string arguments) {
             var finder = new FileFinder(arguments);
 
-            var eval = window.Evaluator as BasePythonReplEvaluator;
-            if (eval != null && eval.CurrentOptions != null) {
-                finder.Search(eval.CurrentOptions.WorkingDirectory);
-                finder.SearchAll(eval.CurrentOptions.SearchPaths, ';');
+            var eval = window.GetPythonEvaluator();
+            if (eval != null) {
+                finder.Search(eval.WorkingDirectory);
+                string searchPaths;
+                // HACK: Should not assume variable name is PYTHONPATH
+                if (eval.EnvironmentVariables.TryGetValue("PYTHONPATH", out searchPaths)) {
```

Comment:
eval.Analyzer.InterpreterFactory.Configuration.PathEnvironmentVariable instead of the hack?